### DISCUSSION TITLE
flink-doris-connector写入日期数据为null

### DISCUSSION THREAD

**Discussion by donsmile** [apache/doris#26517] [A - General / Q&A]
版本：flink-doris-connector-1.17 1.4.0
通过SchemaChange 数据流(JsonDebeziumSchemaSerializer)写入doris库，doris库中的表设计了datetime和date字段，在写入时，拦截了一下数据并解析成json，无论对应字段采用Long型的时间戳，还是java自带的new date等方式，都无法使doris中的datetime和date字段存入值，其他字段正常

**Comment by JNSimba** [comment]
You try to print out the upstream data and see what it is.

**Reply by donsmile** [reply]
I have printed the row data and converted it to json
![image](https://github.com/apache/doris/assets/27986841/70869929-3474-4f00-ac19-60f21419f4ce)

**Comment by donsmile** [comment]
> You try to print out the upstream data and see what it is.



> You try to print out the upstream data and see what it is.



> You try to print out the upstream data and see what it is.
